src/main/resources/application.yml

```diff
@@ -2,13 +2,9 @@ server:
     servlet:
         context-path: /dddsample
 
-cxf:
-    path: /ws
-
 logging:
     level:
-        org.hibernate.SQL: info
-        net.sf.ehcache.config.ConfigurationFactory: error
```

Comment:
Are these lines removed because the classes are no longer used in Spring Boot 3?

Reply:
We don't use it in the project now as you say.